symengine/pow.cpp

```diff
@@ -161,6 +163,14 @@ RCP<const Basic> pow(const RCP<const Basic> &a, const RCP<const Basic> &b)
                 .pow(*rcp_static_cast<const Number>(b));
         }
     }
+    if (is_a<RealDouble>(*a) and is_a<Constant>(*b)) {
```

Comment:
@jppelteret `eval_double(*b)` returns a plain `double`, which I think is not overloaded in the `pow` functions of class`Number` or `Complex`.